st2common/st2common/util/config_loader.py

```diff
@@ -156,8 +156,9 @@ def _assign_default_values(self, schema, config):
             default_value = schema_item.get('default', None)
             is_object = schema_item.get('type', None) == 'object'
             has_properties = schema_item.get('properties', None)
+            config_value = config.get(schema_item_key, None)
```

Comment:
Ah! TBH, I wasn't really looking into assign_default_values logic and didn't realize that config is a simple dictionary, I was just fixing distinction between None and False retvals in conditional. PR updated.